### PR TITLE
[v7] Remove marked

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary

Render `.md` files with MDX instead of Marked. This resulted in a few minor differences in how markdown was rendered (thus the md screenshot changes).

Related to: #294

### Deployment Link

https://terra-dev-site-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 * Breaking
   * Remove the ability to include test evidence, this is covered now by a separate site report.
+  * Remove Use MDX instead of Marked to load markdown files.
```
Comment (emilyrohrbough) [inline CHANGELOG.md:7]:
```suggestion
  * Use MDX instead of Marked to load markdown files.
```

File: config/webpack/plugin/SetupPlugin.js [hunk 2]
```diff
@@ -64,26 +64,11 @@ class SetupPlugin {
...
             ],
             use: [
               babelLoader,
-              {
-                loader: 'devSiteMarked',
-                options: {
-                  baseUrl: this.publicPath,
-                },
-              },
+              mdxLoader,
```
Comment (emilyrohrbough) [inline config/webpack/plugin/SetupPlugin.js:71]: does this improve compile time?

**Review by emilyrohrbough — APPROVED** [review]

File: config/webpack/plugin/SetupPlugin.js (hunk 2 above)
Comment (mjhenkes) [reply config/webpack/plugin/SetupPlugin.js:71]: I wouldn't expect a significant increase, we're trading one loader for another.

Comment (mjhenkes) [reply config/webpack/plugin/SetupPlugin.js:71]: I also didn't collect the before and after.

**Review by StephenEsser — APPROVED** [review]

File: config/webpack/plugin/SetupPlugin.js (hunk 2 above)
Comment (mjhenkes) [reply config/webpack/plugin/SetupPlugin.js:71]: Very Roughly the terra-dev-site `lint jest and compile` stage went from 6 minutes 28 seconds in main to 6 minutes and 21 seconds in the PR branch. So I think thats within tolerances.

**Pull request merged**